src/open_inwoner/openklant/tests/data.py

```diff
@@ -23,15 +26,22 @@
 
 class MockAPIData:
     @classmethod
-    def setUpServices(cls):
-        config = ESuiteKlantConfig.get_solo()
-        config.klanten_service = ServiceFactory(
-            api_root=KLANTEN_ROOT, api_type=APITypes.kc
-        )
-        config.contactmomenten_service = ServiceFactory(
-            api_root=CONTACTMOMENTEN_ROOT, api_type=APITypes.cmc
-        )
-        config.save()
+    def setUpServices(cls, klanten_service_type: str = ""):
```

Comment:
Nitpick, but we can get some extra typing help with

```suggestion
    def setUpServices(cls, klanten_service_type: KlantenServiceType = KlantenServiceType.ESUITE):
```